packages/cli/src/commands/setup/deploy/providers/render.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from 'src/lib'
+
+const RENDER_YAML = `services:
+- type: web
+  name: redwood-api
+  env: node
+  plan: starter
+  buildCommand: yarn && yarn rw prisma migrate deploy && yarn rw build api
+  startCommand: cd api && yarn api-server --functions ./dist/functions
+  envVars:
+  - key: DATABASE_URL
+    fromDatabase:
+      name: redwood-db
+      property: connectionString
+  - key: NODE_VERSION
+    value: 14.16.0
+
+- type: web
+  name: redwood-web
+  env: static
+  buildCommand: yarn && yarn rw build web
+  staticPublishPath: ./web/dist
+  envVars:
+  - key: API_PROXY_HOST
+    fromService:
+      name: redwood-api
+      type: web
+      property: host
+  routes:
+  - type: rewrite
+    source: /*
+    destination: /index.html
+
+databases:
+- name: redwood-db
+`
+const RENDER_HEALTH_CHECK = `
+export const handler = async () => {
+  return {
+    statusCode: 200,
+  }
+}
+`
+const NODE_VERSION = `14.16.0`
+
+// any packages to install
+export const apiPackages = ['@redwoodjs/api-server']
+
+// any files to create
+export const files = [
+  { path: path.join(getPaths().base, 'render.yaml'), content: RENDER_YAML },
+  {
+    path: path.join(getPaths().base, 'api/src/functions/healthz.js'),
```

Comment:
`healthz.js` Unless there's a Render-related need for this naming convention, I'd recommend being descriptive, e.g. `render-health-check...` or maybe adding a comment in the file explaining why the function exists and how it's used (probably a better idea). Understood this should be unique.

Reply:
comment added in `healthz.js`